docs/technologies.md

```diff
@@ -0,0 +1,80 @@
+# Technologies
+
+This document provides a brief overview of some of the key technologies we use in the storage service.
+
+## AWS services
+
+<table>
+  <tr>
+    <td>
+      <img src="./images/Amazon-Simple-Storage-Service_64@5x.png">
+    </td>
+    <td>
+      <strong>S3</strong> – petabyte-scale object storage.
+      We use it for permanent storage of assets, temporary/working storage, and to hold storage "manifests" (our JSON representation of a bag).
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <img src="./images/Arch_Amazon-DynamoDB_64@5x.png">
+    </td>
+    <td>
+      <strong>DynamoDB</strong> – a NoSQL database that we use as a key-value store.
+      We use it to track ingests, store information about versions, and lock around certain processes.
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <img src="./images/Arch_Amazon-Simple-Queue-Service_64@5x.png">
+      <img src="./images/Arch_Amazon-Simple-Notification-Service_64@5x.png">
+    </td>
+    <td>
+      <strong>SNS/SQS</strong> – inter-app message queues.
+      Our apps form a pipeline: an app gets a message from an SQS queue, does some work, then sends an SNS notification to the next app.<br/><br/>
+      We use SNS <em>topics</em> and SQS <em>queues</em>.
+      Queues can subscribe to the output of topics, in a many-to-many relationship.
+      This allows one app to send messages to multiple apps, or one app to receive messages from multiple apps.
+    </td>
```

Comment:
THe structure of this bit seems a bit incongruent with the rest, which seems to follow "Thing - nice simple description of thing. Why we use said thing".

A small image of a pipeline could speak a million words too - I'm sure I've seen one.

Reply:
I'm thinking I might do a whole other bit about messaging tbh, to explain the one-to-many and many-to-one stuff, and also autoscaling. You're right, it's not a good fit here – a "read more" link would be better.

Reply:
How about this? https://github.com/wellcomecollection/storage-service/blob/technologies-doc/docs/inter-app-messaging.md